test/mailers/previews/notifier_preview.rb

```diff
@@ -0,0 +1,7 @@
+class NotifierPreview < ActionMailer::Preview
```

Comment:
@vlasiak, Is that something that being used for debug/testing purpose?  Otherwise please remove.